support-frontend/app/services/AuthenticationService.scala

```diff
@@ -81,6 +81,7 @@ object AsyncAuthenticationService {
     error match {
       // Utilises the custom error introduced in: https://github.com/guardian/identity/pull/1578
       case _: UserCredentialsMissingError => SafeLogger.info(s"unable to authorize user - $error")
+      // Cloud watch alarm based on this log statement.
       case _ => SafeLogger.error(scrub"unable to authorize user", error)
```

Comment:
if you want to have an alarm on a log line, it has to be quite clear, I'd recommend an error code 
```suggestion
      case _ => SafeLogger.error(scrub"AUTH_FAIL: unable to authorize user", error)
```
however @mario-galic had trouble with alarms being lumpy when using this technique of processing logs.  We think this might be because the AWS logs agent didn't always keep up with the logs being produced.
We do have a metric library in support-frontend which would mean it could be explicit and go straight into cloudwatch which I would prefer to do.

Reply:
e.g. https://github.com/guardian/support-frontend/search?q=catalogFailureRequest&unscoped_q=catalogFailureRequest